cypress/views/mcg-ms-common.ts

```diff
@@ -0,0 +1,8 @@
+import { DATA_FEDERATION, DATA_SERVICES, NO_WAIT } from '../constants/common';
+
+export const MCG_MS_Common = {
+  visitMcgMsDashboard: () => {
+    cy.clickNavLink([DATA_SERVICES, DATA_FEDERATION]);
+    cy.location('pathname', NO_WAIT).should('eq', '/mcgms/cluster');
```

Comment:
```suggestion
    cy.location('pathname').should('eq', '/mcgms/cluster');
```
on second thought I would say remove this as well... if we right at one place others writing the test might get urge to add this in their code as well... let's keep code clean from starting only...